Patches.py

```diff
@@ -2172,7 +2176,19 @@ def set_spirit_shortcut(rom, actor_id, actor, scene):
 
     get_actor_list(rom, set_spirit_shortcut)
 
-
+def set_red_ice_transparency(rom):
+    ice_z_coords = (
+        0xF5D8, # z=-2600 Ice Cavern Map Chest
+        0x0283, # z=283   Ice Cavern Compass Chest
+        0xFB80, # z=-1152 Ganons Castle MQ Water Trial Chest
```

Comment:
In my opinion, transparency changes should be generic for all red ice blocks, not just for frozen chests.

Reply:
I agree with this.

Reply:
Done. It now affects all red ice except red ice walls.